False positive with Objects.toString

After https://github.com/typetools/jdk/commit/8b504384bd6f609282f30fa2a5a19c00f8aecaa5, the nullness checker reports the following false positive. Should the first parameter of `Objects.toString` be annotated `@Nullable` instead of `@PolyNull`?

```java
import java.util.Objects;
import org.checkerframework.checker.nullness.qual.Nullable;

class T {
  void f(@Nullable String s) {
    Objects.toString(s, "").toString();
  }
}
```

```
$ ./checker/bin/javac -processor Nullness T.java
...
T.java:6: error: [dereference.of.nullable] dereference of possibly-null reference Objects.toString(s, "")
    Objects.toString(s, "").toString();
```

--- a/checker/tests/nullness/Issue4889.java
+++ b/checker/tests/nullness/Issue4889.java
@@ -1,0 +1,14 @@
+import java.util.Objects;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Issue4889 {
+  void f(@Nullable String s) {
+    Objects.toString(s, "").toString();
+  }
+
+  void g(@Nullable String s) {
+    @NonNull String x = Objects.toString(s, "");
+    @Nullable String y = Objects.toString(s, null);
+  }
+}

